components/common/Dropdown/Dropdown.interfaces.tsx

```diff
@@ -0,0 +1,11 @@
+export interface SelectorItem {
```

Comment:
interfaces should not be in tsx files, use ts files for that

Reply:
Renamed